configs/TUNERCF405/config.h

```diff
@@ -32,6 +32,10 @@
 #define USE_ACC
 #define USE_ACCGYRO_BMI270
 #define USE_MAX7456
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_MAG
+#define USE_MAG_QMC5883
```

Comment:
These sensors are not physical present on [Poly F405](https://www.tunerc.com/product/poly-f405-2s-4s-20a-aio-v1-2/)

```suggestion
```